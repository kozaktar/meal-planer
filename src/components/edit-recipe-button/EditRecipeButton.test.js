import React from 'react'
import {shallow} from 'enzyme'
import {EditRecipeButton} from './EditRecipeButton.component'

describe('AllertPanel test', () => {

    it('EditRecipeButton is rendering without crashing', ()=>{
        expect(shallow(<EditRecipeButton/>)).toMatchSnapshot();
    })
})