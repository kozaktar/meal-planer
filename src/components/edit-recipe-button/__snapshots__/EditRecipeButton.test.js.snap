// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllertPanel test EditRecipeButton is rendering without crashing 1`] = `
<WithStyles(ForwardRef(IconButton))
  aria-label="edit"
  className="makeStyles-addButton-1"
  disableTouchRipple={true}
  size="medium"
>
  <span
    style={
      Object {
        "fontSize": "14px",
        "marginRight": "2px",
      }
    }
  >
    Edit Recipe
  </span>
   
  <EditIcon
    fontSize="default"
  />
</WithStyles(ForwardRef(IconButton))>
`;
